java-vtl-model/src/main/java/no/ssb/vtl/model/VTLNumber.java

```diff
@@ -30,6 +30,8 @@
 
     @Override
     public abstract T get();
+
+    public abstract String toString();
```

Comment:
This conflicts with the toString() Object method. I'd rather have this logic - that is only valid in the case of type conversion - directly into their safeInvoke().

Reply:
Maybe rename it to `asString()`? Then we avoid the conflict.

Reply:
It's better, but it still pulls logic about how to convert this type to another inside the class. We should never use this method except in the VTLFunction implementation. In that sense I don't think it belongs here.